hkm/views/views.py

```diff
@@ -540,9 +540,12 @@ def handle_search(self, request, *args, **kwargs):
             if not self.search_result.get('resultCount') == 0 and 'records' in self.search_result and not kwargs.get(
                     'record'):
                 # Check also if this record is one of user's favorites
+                i = 1
                 for record in self.search_result['records']:
+                    record['index'] = i
                     if favorite_records is not None:
                         record['is_favorite'] = record['id'] in favorite_records
+                    i += 1
```

Comment:
I think you should simplify this by using `enumerate()`, something like:

```
for idx, record in enumerate(self.search_result['records'], 1):
    ...
```
    
See for example https://stackoverflow.com/a/522578

Reply:
You are right, made the change.